client/v1/src/main/java/org/opennms/velocloud/client/v1/VelocloudApiClientV1.java

```diff
@@ -148,4 +158,103 @@ public List<Edge> getEdges(final UUID enterpriseId) throws VelocloudApiException
+
+                    })
+                    .collect(Collectors.toList());
+        } catch (ApiException e) {
+            throw new VelocloudApiException("Error requesting gateways", e);
+        }
+    }
+
+    @Override
+    public List<Enterprise> getEnterprises() throws VelocloudApiException {
+        try {
+            List<EnterpriseProxyGetEnterpriseProxyEnterprisesResultItem> enterprises = this.allApi.enterpriseProxyGetEnterpriseProxyEnterprises(
+                    new EnterpriseProxyGetEnterpriseProxyEnterprises()
+                            .addWithItem(EnterpriseProxyGetEnterpriseProxyEnterprises.WithEnum.EDGES)
+            );
+            return enterprises.stream().map(e ->
+                            Enterprise.builder()
+                                    .setEnterpriseId(UUID.fromString(e.getLogicalId()))
+                                    .setId(e.getId())
+                                    .build())
+                    .collect(Collectors.toList());
+        } catch (ApiException e) {
+            throw new VelocloudApiException("Error requesting enterprises", e);
+        }
+    }
+
+    @Override
+    public List<User> getUsers(final Integer enterpriseId) throws VelocloudApiException {
+        final List<EnterpriseGetEnterpriseUsersResultItem> users;
+        try {
+            users = this.userMaintenanceApi.enterpriseGetEnterpriseUsers(
+                    new EnterpriseGetEnterpriseUsers()
+                            .enterpriseId(enterpriseId));
+            return users.stream().map(user->
+                    User.builder()
+                            .setId(user.getId())
+                            .setUserType(user.getUserType())
+                            .setDomain(user.getDomain())
+                            .setUsername(user.getUsername())
+                            .setFirstName(user.getFirstName())
+                            .setLastName(user.getLastName())
+                            .setEmail(user.getEmail())
+                            .setRoleId(user.getRoleId())
+                            .setRoleName(user.getRoleName())
+                            .setAccessLevel(user.getAccessLevel().getValue())
+                            .setActive(user.getIsActive().getValue() == 0 ?  false : true)
+                            .setLocked(user.getIsLocked().getValue() == 0 ? false : true)
+                            .setNative(user.getIsNative() .getValue()== 0 ? false : true)
+                            .setSshUsername(user.getSshUsername())
+                            .build()
+            ).collect(Collectors.toList());
+        } catch (ApiException e) {
+            throw new VelocloudApiException("ApiException:" + e.getMessage());
+        }
+    }
+
+
+    public static InetAddress getValidInet4Address(String ip) {
+        try {
+            InetAddress address = InetAddress.getByName(ip);
+            if (address.getHostAddress().equals(ip)) {
```

Comment:
AFAIR, there is a helper in guava already. In addition, `getByName` tries to resolve the name. Which we should avoid, as it could put some load on the resolver.